torch/_dynamo/output_graph.py

```diff
@@ -1381,6 +1381,16 @@ def compile_and_call_fx_graph(self, tx, rv, root, replaced_outputs):
             with self.restore_global_state():
                 compiled_fn = self.call_user_compiler(gm)
 
+            torch._logging.trace_structured(
```

Comment:
Any reason not to use encoding: json here? That way you could return json.dumps() and have slightly more structured data.
 
Example usage:
https://github.com/pytorch/pytorch/blob/b0be30dd7912797dcdbe01c2b1186f45482813ff/torch/_functorch/_aot_autograd/autograd_cache.py#L362-L369

Reply:
I was just cargo-culting, this definitely seems better - will update!